.github/workflows/build.yml

```diff
@@ -226,6 +263,7 @@ jobs:
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
           TERRARIA_VERSION: 1436
           TMLVERSION: ${{ steps.version.outputs.new_version }}
+          LTSVERSION: ${{ steps.lts.outputs.lts_version }}
```

Comment:
Love this, I was just going to hard-code it, but this is great

Reply:
Yup, tried to make it more maintainable. Don't want to have to update this every month.